arcgis-ios-sdk-samples/Geometry/Format coordinates/FormatCoordinates.storyboard

```diff
@@ -1,168 +1,37 @@
-                                            <constraint firstAttribute="bottom" secondItem="f0e-3N-1ay" secondAttribute="bottom" id="Uuc-kp-abY"/>
-                                            <constraint firstAttribute="height" constant="151" id="Z4H-mT-Fo4"/>
-                                            <constraint firstAttribute="height" constant="233" id="er3-ni-T2e"/>
-                                            <constraint firstAttribute="width" constant="250" id="sm8-7f-Fjg"/>
-                                            <constraint firstItem="f0e-3N-1ay" firstAttribute="top" secondItem="13U-3h-0zt" secondAttribute="top" id="ysb-Np-CVo"/>
+                                            <constraint firstAttribute="height" constant="300" id="NX2-ML-r3F"/>
                                         </constraints>
-                                        <variation key="default">
-                                            <mask key="constraints">
-                                                <exclude reference="Z4H-mT-Fo4"/>
-                                                <exclude reference="sm8-7f-Fjg"/>
-                                            </mask>
-                                        </variation>
-                                        <variation key="heightClass=compact">
-                                            <mask key="constraints">
-                                                <exclude reference="er3-ni-T2e"/>
-                                                <include reference="sm8-7f-Fjg"/>
-                                            </mask>
-                                        </variation>
-                                        <variation key="heightClass=regular-widthClass=regular">
-                                            <mask key="constraints">
-                                                <include reference="Z4H-mT-Fo4"/>
-                                                <exclude reference="er3-ni-T2e"/>
-                                            </mask>
-                                        </variation>
-                                    </scrollView>
+                                        <connections>
+                                            <segue destination="geL-f0-eoD" kind="embed" id="fDz-l6-wCh"/>
+                                        </connections>
+                                    </containerView>
                                     <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="vLE-Mm-bqf" customClass="AGSMapView">
-                                        <rect key="frame" x="0.0" y="233" width="375" height="414"/>
+                                        <rect key="frame" x="0.0" y="300" width="375" height="347"/>
                                     </view>
                                 </subviews>
                                 <variation key="heightClass=compact" axis="horizontal"/>
```

Comment:
I would suggest using regular width, compact height for the variant. That way the stack view uses a horizontal distribution on iPad, which I think is a bit more natural.

Reply:
I don't know what the variant even is.

Reply:
A variation allows setting a different value for a property in a given size class. This is what it looks like:

![screen shot 2018-12-03 at 3 54 41 pm](https://user-images.githubusercontent.com/2257493/49409035-c8329e80-f713-11e8-8550-21cec1a45519.png)

And that's the combination I would recommend. In my experience, it is best to use side-by-side everywhere except compact width, regular height environments.

Reply:
Thanks for that explanation. I updated it to your recommendation. This PR should be good to go.